lib/Graph/Graph.cpp

```diff
@@ -1397,7 +1397,8 @@ QuantizeNode *Function::createQuantize(llvm::StringRef name, NodeValue input,
                                        TypeRef outTy) {
   assert(input.getElementType() == ElemKind::FloatTy &&
          "Input must be a floating type");
-  assert(outTy->getElementType() == ElemKind::Int8QTy &&
+  assert((outTy->getElementType() == ElemKind::Int8QTy ||
+          outTy->getElementType() == ElemKind::Int32QTy) &&
```

Comment:
In case of int32q do we properly merge quantization into constant node? like Quantize(Value) -> QuantizedValue and eliminate `Quantize` operation completely